packages/patternfly-4/react-core/src/components/Dropdown/DropdownItem.js

```diff
@@ -25,28 +28,68 @@ const defaultProps = {
   isHovered: false,
   component: 'a',
   isDisabled: false,
-  href: '#'
+  href: '#',
+  keyHandler: () => undefined,
+  index: -1,
+  sendRef: () => undefined
 };
 
-const DropdownItem = ({ className, children, isHovered, component: Component, isDisabled, ...props }) => {
-  const additionalProps = props;
-  if (Component === 'a') {
-    additionalProps['aria-disabled'] = isDisabled;
-    additionalProps.tabIndex = isDisabled ? -1 : additionalProps.tabIndex;
-  } else if (Component === 'button') {
-    additionalProps.disabled = isDisabled;
+class DropdownItem extends React.Component {
+  constructor(props) {
+    super(props);
+    this.ref = React.createRef();
+    this.onKeyDown = this.onKeyDown.bind(this);
```

Comment:
Same with this. If it is declared as a class property using the arrow notation there is no need to bind here.